.github/workflows/watchdog.yml

```
@@ -0,0 +1,120 @@
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  pr-description:
+    name: PR has description
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR body
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)